packages/forms-web-app/src/views/projects/examination.njk

```diff
@@ -66,6 +66,26 @@
     govuk-section-break--xl
     govuk-section-break--visible"/>
 
+ <h2 class="govuk-heading-m">Contact us for support</h2>
+          <p class="govuk-body"> 
+            <strong>
+              Project case team email:
+            </strong>
+             </br>
+                <a class="govuk-link" href="mailto:{{ appData.ProjectEmailAddress }}">{{ appData.ProjectEmailAddress }}
```

Comment:
This `<a>` tag doesn't need to be indented (`</br>` is self-closing so the `<a>` isn't nested within it)